synapse/lib/membrane.py

```diff
@@ -1,32 +0,0 @@
-import synapse.eventbus as s_eventbus
-
-import synapse.lib.auth as s_auth
-
-class Membrane(s_eventbus.EventBus):
```

Comment:
@vEpiphyte @invisig0th - I removed Membrane in this PR because you can achieve the same thing by creating an auth `Role`, adding `rule`s to it, and calling `allowed` on the `Role`.